wormchain/x/tokenfactory/types/codec.go

```diff
@@ -0,0 +1,48 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	// this line is used by starport scaffolding # 1
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+func RegisterCodec(cdc *codec.LegacyAmino) {
+	cdc.RegisterConcrete(&MsgCreateDenom{}, "osmosis/tokenfactory/create-denom", nil)
+	cdc.RegisterConcrete(&MsgMint{}, "osmosis/tokenfactory/mint", nil)
+	cdc.RegisterConcrete(&MsgBurn{}, "osmosis/tokenfactory/burn", nil)
+	cdc.RegisterConcrete(&MsgForceTransfer{}, "osmosis/tokenfactory/force-transfer", nil)
+	cdc.RegisterConcrete(&MsgChangeAdmin{}, "osmosis/tokenfactory/change-admin", nil)
+}
```

Comment:
Should we rename these to `"wormhole/tokenfactory/..."`?

Reply:
I think we could, but since those types are registered with osmosis, it makes sense to me to keep osmosis unless we change the package name too. And the chance of the current package name staying the same is probably greatest when there is a tokenfactory module we can import in the future. That would keep some consistency.